src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -950,7 +952,21 @@ void Graph::PullOutputData(BlobMap &out) {
             if (expectedDesc.getLayout() == InferenceEngine::Layout::BLOCKED) {
                 expectedDesc = TensorDesc(expectedDesc.getPrecision(), expectedDesc.getLayout());
             }
-            out[name]->setShape(outDims);
+
+            if (std::count(inplacedOutPorts.begin(), inplacedOutPorts.end(), name) &&
```

Comment:
Sync up InferRequest output with resized Graph output.

The old value is destroyed as it is unreferenced. Graph output is switched to InferRequest.

Probably we need recover Graph output to its previous memory (and memory manager)... in case it serves another infer_request which cannot changeDefaultPtr.